helm/app-admission-controller/templates/service.yaml

```diff
@@ -5,10 +5,6 @@ metadata:
   namespace: {{ include "resource.default.namespace" . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
-  annotations:
-    prometheus.io/scrape: "true"
-    prometheus.io/path: "/metrics"
-    prometheus.io/port: "8080"
 spec:
   ports:
   - port: 443
```

Comment:
We need to export port 8080 otherwise the service monitor will not be able to scrape